front-end/src/components/modal/create/CreateGraphModal/CreateGraphModal.jsx

```diff
@@ -42,17 +42,14 @@ export const CreateGraphModal = ({
   useEffect(() => {
     // Fetch data when the component mounts
     ApiUtil.getFolder('csv')
-      .then((response) => response.json())
```

Comment:
Is this something that you changed that makes this work now or would it have always worked to remove these?

Reply:
ApiUtils now returns the parsed response via 'response.json()' or similar.